web-api/elasticsearch/efcms-docket-entry-mappings.js

```diff
@@ -26,10 +26,22 @@ module.exports = {
     },
     'documentContents.S': {
       analyzer: 'ustc_analyzer',
+      fields: {
+        exact: {
+          analyzer: 'english_exact',
+          type: 'text',
+        },
+      },
       type: 'text',
     },
     'documentTitle.S': {
       analyzer: 'ustc_analyzer',
+      fields: {
+        exact: {
+          analyzer: 'english_exact',
+          type: 'text',
+        },
+      },
```

Comment:
Anything special needed to deploy these mapping changes?

Reply:
It'll need an ElasticSearch re-index, which is required by the migrations.